conda_package_handling/cli.py

```diff
@@ -41,6 +41,25 @@ def parse_args(parse_this=None):
     convert_parser.add_argument('out_ext', help="extension of file to convert to.  "
                                 "Examples: .tar.bz2, .conda")
     convert_parser.add_argument("--out-folder", help="Folder to dump final archive to")
+
+    gpg_sign_parser = sp.add_parser('gpg-sign', help="sign package using GPG", aliases=['gpgs'])
+    gpg_sign_parser.add_argument('--tool', help="GPG program to be used to sign archives",
+                                 default='gpg', dest='gpgs_tool')
+    gpg_sign_parser.add_argument('--identity', help="GPG identity to be used to sign archives",
+                                 dest='gpgs_identity')
+    gpg_sign_parser.add_argument('--homedir', help="GPG home directory", dest='gpgs_homedir')
+    gpg_sign_parser.add_argument('archives', help="Path to archive(s) which need(s) to be signed. "
+                                 "Glob patterns accepted")
+    gpg_sign_parser.add_argument("--out-folder", help="Folder to dump signed archive(s) to")
+
+    gpg_verify_parser = sp.add_parser('gpg-verify', help="verify package using GPG",
+                                      aliases=['gpgv'])
```

Comment:
I couldn't think of better short aliases other than 'gpgs' and 'gpgv'. Suggestions welcome!